var/spack/repos/builtin/packages/py-bitstring/package.py

```diff
@@ -10,9 +10,15 @@ class PyBitstring(PythonPackage):
     """Simple construction, analysis and modification of binary data."""
 
     homepage = "http://pythonhosted.org/bitstring"
-    pypi = "bitstring/bitstring-3.1.5.zip"
+    pypi = "bitstring/bitstring-3.1.6.tar.gz"
 
+    version("4.0.2", sha256="a391db8828ac4485dd5ce72c80b27ebac3e7b989631359959e310cd9729723b2")
     version("3.1.5", sha256="c163a86fcef377c314690051885d86b47419e3e1770990c212e16723c1c08faa")
 
-    # pip silently replaces distutils with setuptools
     depends_on("py-setuptools", type="build")
+    depends_on("py-bitarray@2.7.4", when="@4.1:")
```

Comment:
Where did you find this dep?

Reply:
Sorry, should have said in the PR comment. They've switching over to use `bitstring` as of `@4.1:`: https://github.com/scott-griffiths/bitstring#41-beta-now-available. That's not the version I'm adding as it's still in beta (hence `when="@4.1:"`), but I was just including it now for completeness. The `main` branch has this dep in its `pyproject.toml`.

Reply:
Can remove it for now if you prefer...

Reply:
Gotcha, didn't realize it was for future versions